pycon-ukraine-2014/videos/pycon-ukraine-2014-lightning-talks-day-2.json

```diff
@@ -5,7 +5,7 @@
   "description": "",
   "duration": null,
   "id": 3354,
-  "language": "ukr",
+  "language": "ru",
```

Comment:
the language is string in ISO_639-3 format.

for Russian it is "rus"

here are examples of such codes: https://github.com/pyvideo/pyvideo_scrape/blob/master/events.yml